WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsLiveSiteInTestModeView.swift

```diff
@@ -23,24 +25,24 @@ struct InPersonPaymentsLiveSiteInTestMode: View {
 
 private enum Localization {
     static let title = NSLocalizedString(
-        "WooCommerce Payments is in Test Mode",
-        comment: "Title for the error screen when WooCommerce Payments is in test mode on a live site"
+        "%1$@ is in Test Mode",
+        comment: "Title for the error screen when a card present payments plugin is in test mode on a live site"
```

Comment:
It's always helpful for translators to explicitly call out what will be in the placeholder
```suggestion
        comment: "Title for the error screen when a card present payments plugin is in test mode on a live site. %1$@ is a placeholder for the plugin name, e.g. WooCommerce Stripe Gateway"
```